tox.ini

```
@@ -165,8 +165,7 @@ envlist =
     python-template_jinja2-py37-jinja2030103,
     python-template_mako-{py37,py38,py39,py310,py311,py312,py313},
     rabbitmq-messagebroker_pika-{py37,py38,py39,py310,py311,py312,py313,pypy310}-pikalatest,
-    ;; Comment out Kombu until we can root cause the crash.
-    ; rabbitmq-messagebroker_kombu-{py38,py39,py310,py311,py312,py313,pypy310}-kombulatest,
+    rabbitmq-messagebroker_kombu-{py38,py39,py310,py311,py312,py313,pypy310}-{kombulatest,kombu050204},
```

```suggestion
    rabbitmq-messagebroker_kombu-{py38,py39,py310,py311,py312,py313,pypy310}-kombulatest,
    rabbitmq-messagebroker_kombu-{py38,py39,py310,py311,pypy310}-kombu050204,
```

Python 3.11 and higher were unsupported on this version, but seems it works fine on Python 3.11. 3.12 and 3.13 are failing so we should remove those.